bidirectional_character_scan/repository-scanner.yml

```diff
@@ -0,0 +1,30 @@
+name: Scan Actions
```

Comment:
This looks like a duplicate of `repository-scanner.yml` above.

Reply:
removed